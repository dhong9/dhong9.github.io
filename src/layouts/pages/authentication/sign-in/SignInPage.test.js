// Unit test libraries
import React from "react";
import renderer from "react-test-renderer";

// Component to test
import SignInPage from "layouts/pages/authentication/sign-in";

// Mocks
jest.mock("pages/LandingPages/SignIn", () => {
  const { forwardRef } = jest.requireActual("react");
  return {
    __esModule: true,
    default: forwardRef(() => <div>SignIn</div>),
  };
});

describe("SignInPage", () => {
  it("renders", () => {
    const component = renderer.create(<SignInPage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
