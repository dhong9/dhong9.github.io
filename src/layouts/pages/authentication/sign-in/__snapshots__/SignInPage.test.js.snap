// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInPage renders 1`] = `
<div>
  SignIn
</div>
`;
